bot/cogs/antimalware.py

```diff
@@ -17,37 +18,32 @@ def __init__(self, bot: Bot):
     @Cog.listener()
     async def on_message(self, message: Message) -> None:
         """Identify messages with prohibited attachments."""
-        rejected_attachments = False
-        detected_pyfile = False
+        if len(message.attachments) == 0:
+            return
+
+        embed = Embed()
         for attachment in message.attachments:
             if attachment.filename.lower().endswith('.py'):
```

Comment:
since `attachment.filename.lower()` is used twice maybe it can be refactored to the top of the loop here to reduce clutter?
```suggestion
            filename = attachment.filename.lower()
            if filename.endswith('.py'):
```